core/include/usBundleEvent.h

```diff
@@ -108,7 +108,32 @@ class US_Core_EXPORT BundleEvent
      * <p>
      * The bundle has been removed from the Framework.
      */
-    UNINSTALLED
+    UNINSTALLED,
```

Comment:
I think its worthwhile to assign these enums the same values as defined in the OSGi core spec.
I did this with ```FrameworkEvents```.